sktalk/corpus/parsing/cha.py

```diff
@@ -0,0 +1,69 @@
+import re
+import pylangacq
+from ..conversation import Conversation
+from ..utterance import Utterance
+from .parser import Parser
+
+
+class ChaParser(Parser):
+    def parse(self, file):
+        """Parse conversation file in Chat format
+
+        Args:
+            file (str): path to file in .cha format
+
+        Returns:
+            Conversation: A Conversation object representing the conversation in the file.
+        """
+        chatfile = pylangacq.read_chat(file)
+        chat_utterances = chatfile.utterances(by_files=False)
+
+        metadata = ChaParser._get_metadata(chatfile)
+
+        utterances = [ChaParser._to_utterance(
+            row, file) for row in chat_utterances]
+
+        return Conversation(utterances, metadata)
+
+    @staticmethod
+    def _to_utterance(row, file):
```

Comment:
I am not sure I understand the purpose of this function. What is the role of the `row` here?
My impression is that this should be a method of the `Conversation` class which would allow a logic like this:

```
my_conversation = ChaParser.parse(file)
my_utterance = my_conversation.utterances(row)
```
This assumes that `row` is an index or some similar pointer, but I am not sure about that.